src/components.typ

```diff
@@ -113,7 +113,7 @@
 /// That means that `#checkerboard[...][...]` stacks horizontally and `#checkerboard(columns: 1)[...][...]` stacks vertically.
 ///
 /// -> content
-#let checkerboard(columns: none, rows: none, ..bodies) = {
+#let checkerboard(columns: none, rows: none, alignment: center + horizon, color1: white, color2: silver, ..bodies) = {
```

Comment:
Maybe we can use primary-color and secondary-color as names

Reply:
I changed it to simply `primary` and `secondary` following the example of `progress-bar`, `simple-navigation` and `knob-marker`. I hope this is OK!